common4j/src/main/com/microsoft/identity/common/java/opentelemetry/AttributeName.java

```diff
@@ -333,5 +333,10 @@ public enum AttributeName {
     /**
      * Records the stacktrace for an out-of-memory exception.
      */
-    out_of_memory_exception_stacktrace
+    out_of_memory_exception_stacktrace,
+
+    /**
+     * Records if an IntuneAppProtectionPolicyRequiredException is received as part of this request.
+     */
+    intune_protection_policy_exception_present
```

Comment:
I don't think we want a separate field for this and `enrollment_id_present`. We discussed in scrum that these 2 should essentially be a single field called as `is_mam_flow` and emit it as true if either we get an IntuneAppProptectionPolicyException OR we have an enrollment id.

Reply:
Gotcha, we can do that. I figured having the separation could be useful, and when we use an MV or query we can use an or expression to check for both fields, but i can create just one for simplicity